actionview/lib/action_view/helpers/tags/base.rb

```diff
@@ -149,7 +149,7 @@ def select_content_tag(option_tags, options, html_options)
             end
 
             value = options.fetch(:selected) { value(object) }
-            select = content_tag("select", add_options(option_tags, options, value), html_options)
+            select = content_tag("select", add_options(option_tags, options, value), html_options.except!("skip_default_ids", "allow_method_names_outside_object"))
```

Comment:
Is not this being deleted in the initializer? Instead of removing here we should never allow it to reach this place.